docs/user/user/pings/data-control-plane/experimenter-configuration.md

```diff
@@ -0,0 +1,21 @@
+# Experimenter Configuration
+
+The structure of this configuration is a key-value collection with the name of the Glean ping serving as the key and the values are booleans representing whether the ping is enabled (`true`) or not (`false`).
```

Comment:
Grammar nit: plural disagreement "ping serving as the key" "values are booleans"